src/pages/Integrations.vue

```diff
@@ -0,0 +1,60 @@
+<router lang="yaml">
+  name: WSL Integration
+</router>
+
+<script lang="ts">
+import os from 'os';
+import { ipcRenderer } from 'electron';
+import Vue from 'vue';
+import Integration from '@/components/Integration.vue';
+
+export default Vue.extend({
+  components: { Integration },
+  data() {
+    return {
+      /** @type Record<string, boolean | string> */
+      integrations: {},
+    };
+  },
+  computed:   {
+    hasIntegration() {
+      return os.platform().startsWith('win');
+    },
```

Comment:
This is now irrelevant (the whole page only shows on Windows, so this will always be true).